ocs_ci/cleanup/aws/defaults.py

```diff
@@ -2,7 +2,7 @@
 Defaults module for AWS cleanup
 """
 AWS_REGION = 'us-east-2'
-CLUSTER_PREFIXES_TO_EXCLUDE_FROM_DELETION = {'jnk': '36'}
-MINIMUM_CLUSTER_RUNNING_TIME_FOR_DELETION = 10
+CLUSTER_PREFIXES_SPECIAL_RULES = {'jnk': 36, 'dnd': 36}
```

Comment:
DND is stand for do not delete so I am curious if this should be really treated like we don't want to delete such resources at all.  @RazTamir @ebenahar  thoughts?

Reply:
I think LR (long running) is better wording

Reply:
we can have more such prefixes for dnd we can have some high number like 1000, for `LR` actually `lr` we can have other timeframe defined like 10 days for example? Or not sure what long running time frame should be?

I guess we cannot use capital letter as it's not propagated to AWS instance names and DNS record so I would stick with small letters . 

Also not sure if cluster name can start with the number, we can have defined later on also something like s5d - Stay 5 days.

Reply:
Also not sure if cluster name can start with the number, we can have defined later on also something like s5d - Stay 5 days. -> this sounds like the best option. We will need also to read this name and translate it to the number of hours this cluster should remain running.

Reply:
@ebenahar all we need to do is include the rule in this dict, we won't need to do any evaluation of the prefix past checking if the cluster name starts with it. I think using `lr` rather than `s` is cleared as well, and we can incorporate the number of days that we wish to keep this long running cluster up. For example:

```python
RULES = {'jnk': 36, 'DND': never, 'LR1': 24, 'LR2': 48, 'LR3': 72, 'LR4': 96, 'LR5': 120}